### PR TITLE
[SLM] Update StableLM model and migrate it to paged KV Cache

### PR DESCRIPTION
**Pull request by tlopex**
Due to the [update of stableLM](https://huggingface.co/stabilityai/stablelm-zephyr-3b/commit/8b471c751c0e78cb46cf9f47738dd0eb45392071)
I modified part of model accordingly.
Result screenshot:
图片

In addition, I would like to modify the code and have this model migrated to the new PagedKVCache in few days after this pr is merged.

### CODE REVIEW

File: python/mlc_chat/model/stable_lm/stablelm_model.py [hunk 1]
```diff
@@ -19,16 +19,16 @@
 
 
 @dataclasses.dataclass
-class StableLMEpochConfig(ConfigBase):  # pylint: disable=too-many-instance-attributes
+class StableLmConfig(ConfigBase):  # pylint: disable=too-many-instance-attributes
```
Comment (tqchen) [inline python/mlc_chat/model/stable_lm/stablelm_model.py:22]: StableLM

**Comment by tqchen** [comment]
@tlopex can you directly update to PagedKV? we would like to finish these items soon

File: python/mlc_chat/model/stable_lm/stablelm_model.py (hunk 1 above)
Comment (tlopex) [reply python/mlc_chat/model/stable_lm/stablelm_model.py:22]: StableLm is correct according to it's update.

**Comment by tlopex** [comment]
@tqchen Okay, I'll do it soon.

**Comment by tlopex** [comment]
The model have been migrated.
图片

**Comment by tqchen** [comment]
@tlopex can you help to validate on serving layer as well?

**Comment by tlopex** [comment]
Here is the test of MLC serve of StableLM.

My test_server_rest_api.py
```
# test_server_rest_api.py
import requests
import json

model = "dist/stablelm-zephyr-3b-q4f16_1-MLC"
port = 8000
payload = {
    "model": f"{model}",
    "messages": [{"role": "user", "content": "Can you tell me a joke of an ancient king?"}],
    "stream": False,
}

r = requests.post(f"http://127.0.0.1:{port}/v1/chat/completions", json=payload)
if r.status_code != 200:
    print(r.json())
else:
    print(r.json()["choices"][0]["message"]["content"])
```

The response:
```
tlopex@tlopex-OMEN-by-HP-Laptop-17-ck1xxx:~/mlc-llm$ python test_server_rest_api.py 
Sure! Here's a funny ancient king joke for you:

A king once asked his doctor, "Doc, what can I do to improve the health of my soldiers?"

The doctor replied, "Shorter, quicker, and on less meat."

The king laughed, and his soldiers laughed, and his subjects laughed, but no one knew why.. This went on for years until finally a wise man said, "I know why the king made his soldiers shorter, quicker, and on less meat! He wanted to branch out his army!"
```
MLC serve running and status:
```
tlopex@tlopex-OMEN-by-HP-Laptop-17-ck1xxx:~/mlc-llm$ python -m mlc_chat.serve.server --model dist/stablelm-zephyr-3b-q4f16_1-MLC --model-lib-path ./dist/libs/stablelm-zephyr-3b-q4f16_1-cuda.so
[2024-03-11 21:30:06] INFO auto_device.py:76: Found device: cuda:0
[2024-03-11 21:30:07] INFO auto_device.py:85: Not found device: rocm:0
[2024-03-11 21:30:07] INFO auto_device.py:85: Not found device: metal:0
[2024-03-11 21:30:08] INFO auto_device.py:85: Not found device: vulkan:0
[2024-03-11 21:30:09] INFO auto_device.py:85: Not found device: opencl:0
[2024-03-11 21:30:09] INFO auto_device.py:33: Using device: cuda:0
[2024-03-11 21:30:09] INFO chat_module.py:373: Using model folder: /home/tlopex/mlc-llm/dist/stablelm-zephyr-3b-q4f16_1-MLC
[2024-03-11 21:30:09] INFO chat_module.py:374: Using mlc chat config: /home/tlopex/mlc-llm/dist/stablelm-zephyr-3b-q4f16_1-MLC/mlc-chat-config.json
[2024-03-11 21:30:09] INFO chat_module.py:516: Using library model: ./dist/libs/stablelm-zephyr-3b-q4f16_1-cuda.so
[2024-03-11 21:30:10] INFO engine.py:240: Estimated KVCacheConfig "max_total_sequence_length": 37649.
[2024-03-11 21:30:10] INFO engine.py:245: Estimated total single GPU memory usage: 14504.74 MB (Parameters: 1500.05 MB. KVCache: 11894.40 MB. Temporary buffer: 1110.29 MB)
INFO:     Started server process [3835]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
INFO:     127.0.0.1:46258 - "POST /v1/chat/completions HTTP/1.1" 200 OK
```

**Pull request merged**